src/gsSolver/gsKroneckerOp.h

```diff
@@ -0,0 +1,87 @@
+/** @file gsKroneckerOp.h
+
+    @brief Provides a linear operator representing the Kronecker product of linear operators
+
+    This file is part of the G+Smo library.
+
+    This Source Code Form is subject to the terms of the Mozilla Public
+    License, v. 2.0. If a copy of the MPL was not distributed with this
+    file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+    Author(s): C. Hofreither, S. Takacs
+*/
+#pragma once
+
+#include <gsCore/gsExport.h>
+#include <gsSolver/gsLinearOperator.h>
+
+namespace gismo
+{
+
+/// @brief Class for representing a Kronecker product of operators of type \a gsLinearOperator
+///
+/// \ingroup Solver
+template <class T>
+class gsKroneckerOp : public gsLinearOperator<T>
+{
+    typedef typename gsLinearOperator<T>::Ptr BasePtr;
+public:
+
+    /// Shared pointer for gsKroneckerOp
+    typedef memory::shared_ptr<gsKroneckerOp> Ptr;
+
+    /// Unique pointer for gsKroneckerOp
+    typedef memory::unique_ptr<gsKroneckerOp> uPtr;
+
+    /// Kronecker product of a given list of operators
+    gsKroneckerOp(const std::vector< BasePtr >& ops)
```

Comment:
const std::vector< BasePtr >& ops  ---> std::vector< BasePtr > ops 
and m_ops(give(ops))